src/test/java/uk/gov/justice/digital/hmpps/keyworker/services/health/IntegrationTest.kt

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.digital.hmpps.keyworker.services.health
+
+import com.amazonaws.services.sqs.AmazonSQS
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.BeforeEach
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.mock.mockito.SpyBean
+import org.springframework.boot.test.web.client.TestRestTemplate
+import org.springframework.security.authentication.TestingAuthenticationToken
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.test.context.ActiveProfiles
+import uk.gov.justice.digital.hmpps.whereabouts.integration.wiremock.EliteMockServer
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
+abstract class IntegrationTest {
+  @Autowired
+  lateinit var restTemplate: TestRestTemplate
+
+  @SpyBean
+  @Qualifier("awsSqsClient")
+  internal lateinit var awsSqsClient: AmazonSQS
+
+  companion object {
+    internal val eliteMockServer = EliteMockServer()
+
+    @BeforeAll
+    @JvmStatic
+    fun startMocks() {
+      eliteMockServer.start()
```

Comment:
junit 5 doesn't have class rules, so have to do it manually.  They haven't got around to adding support to wiremock for junit 5 yet either